tests/acceptance/README.md

```diff
@@ -0,0 +1,38 @@
+Acceptance tests for Mender
+===========================
+
+
+Prerequisites
+-------------
+
+* PyTest
+* Fabric (fabfile.org)
+
+
+Running
+-------
+
+To run these tests you need to be in this directory, and the meta-mender-qemu
+layer must be next to the meta-mender layer (this layer). In addition, there
+must be a symlink "image.dat" pointing to the image you want to test. Normally
```

Comment:
Maybe: In addition you must provide ```image.dat``` symlink pointing to image you want to test.